packages/devtools_app/test/cpu_profile_model_test.dart

```diff
@@ -4,6 +4,8 @@
 
 // @dart=2.9
 
+// ignore_for_file: avoid_redundant_argument_values
```

Comment:
What's this for?

Reply:
This lint can be triggered for files that have not been migrated to null safety that are importing null safe files. Once this file is migrated, the ignore can be removed.